vista_2d/vista_2d_tutorial_monai.ipynb

```diff
@@ -0,0 +1,1405 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #236.    # Loss plot visualization 
</pre><p>Would suggest add this part to a separate cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Agreed</p>

Reply:
Sure.